testkit/src/main/java/org/apache/calcite/test/SqlOperatorTest.java

```diff
@@ -13686,6 +13673,31 @@ void testCastTruncates(CastType castType, SqlOperatorFixture f) {
     }
   }
 
+  static class AliasInfo {
+    private final SqlFunction function;
+    private final String alias;
+    private final SqlLibrary[] libraries;
```

Comment:
Constructor parameter should be varargs, not array.

`libraries` field should be immutable list, not array.